actors/builtin/miner/miner_actor.go

```diff
@@ -485,9 +485,16 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 
 	store := adt.AsStore(rt)
 	var st State
+	var err error
 	newlyVested := big.Zero()
+	feeDebtPenalty := abi.NewTokenAmount(0)
 	rt.State().Transaction(&st, func() {
-		verifyPledgeMeetsInitialRequirements(rt, &st)
+		newlyVested, err = st.UnlockVestedFunds(store, rt.CurrEpoch())
+		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to vest funds")
+		unlockedBalance := st.GetUnlockedBalance(rt.CurrentBalance())
```

Comment:
Nit: Move this next to line 497, after the fee debt is set to zero. I think it would be clearer as

```
feeDebtPenalty := verifyPledgeRequirementsAndRepayDebts
unlockedBalance := st.GetUnlockedBalance(rt.CurrentBalance() - feeDebtPenalty)
```

Blocking: It seems wrong to me that this is using unlocked, rather than available balance. Balance already covering IP cannot be used here. If it's right, please explain in comments.